src/draw/sw/lv_draw_sw_letter.c

```
@@ -98,13 +98,20 @@ void lv_draw_sw_letter(lv_draw_ctx_t * draw_ctx, const lv_draw_label_dsc_t * dsc
     lv_font_glyph_dsc_t g;
     bool g_ret = lv_font_get_glyph_dsc(dsc->font, &g, letter, '\0');
     if(g_ret == false)  {
-        /*Add warning if the dsc is not found
-         *but do not print warning for non printable ASCII chars (e.g. '\n')*/
-        if(letter >= 0x20 &&
-           letter != 0xf8ff && /*LV_SYMBOL_DUMMY*/
-           letter != 0x200c) { /*ZERO WIDTH NON-JOINER*/
```

I think these non-printable characters should silently ignored